packages/components/src/components/field/field.jsx

```diff
@@ -5,8 +5,8 @@ import React from 'react';
 const Field = ({ message, className, type }) => (
     <p
         className={classNames('dc-field', className, {
-            'dc-field__error': type === 'error',
-            'dc-field__warn': type === 'warn',
+            'dc-field-error': type === 'error',
```

Comment:
should it be `--error` and `--warn` instead?

Reply:
Agreed.